parts/swarm/swarmmastervars.t

```diff
@@ -97,7 +97,7 @@
       ]
     ],
 {{if .OrchestratorProfile.IsSwarmMode}}
-    "orchestratorName": "swarm",
+    "orchestratorName": "swarmm",
```

Comment:
this is only used to name resources, can we make it swarrmmode for clarity?

Reply:
"swarmm" just looks like a spelling mistake which is what tricked you into changing it the first time

Reply:
Preference for not making a change